.github/workflows/lint-type-safe.yml

```
@@ -0,0 +1,42 @@
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Install pnpm
+        uses: pnpm/action-setup@v4
+        with:
+          version: '10'
+
+      - name: Use Node.js ${{ matrix.node-version }}
```

actionlint (1.7.8)

property "node-version" is not defined in object type {}

(expression)